src/components/feedback-message/README.md

```diff
@@ -20,13 +20,12 @@ import { FeedbackMessage } from '@nomios/web-uikit';
 
 | name | type | default | description |
 | ---- | ---- | ------- | ----------- |
-| children | string | *required* | Text to render. |
+| children | node | *required* | The contents to render, usually a text |
```

Comment:
It seems weird to me that a `feedback-message` component can get other than text as children. But I think I got your point.